jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenter.java

```diff
@@ -419,7 +420,7 @@ public void openProcessInstanceView(final String processInstanceId) {
 
     public interface TaskListView<T extends AbstractTaskListPresenter> extends MultiGridView<TaskSummary, T> {
 
-        void addDomainSpecifColumns(ListTable<TaskSummary> extendedPagedTable,
+        void addDomainSpecifColumns(ExtendedPagedTable<TaskSummary> extendedPagedTable,
```

Comment:
is this change necessary?

Reply:
yes, AbstractMultiGridView's method public void addDomainSpecifColumns(ExtendedPagedTable<T> extendedPagedTable,
                                       Set<String> columns) .